.github/workflows/license.yaml

```diff
@@ -1,38 +0,0 @@
-# Licensed to SkyAPM org under one or more contributor
-# license agreements. See the NOTICE file distributed with
-# this work for additional information regarding copyright
-# ownership. SkyAPM org licenses this file to you under
-# the Apache License, Version 2.0 (the "License"); you may
-# not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing,
-# software distributed under the License is distributed on an
-# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-# KIND, either express or implied.  See the License for the
-# specific language governing permissions and limitations
-# under the License.
-
-name: License
-
-on:
-  pull_request:
-  push:
-    branches:
-      - master
-
-jobs:
-  License:
-    runs-on: ubuntu-latest
-    steps:
-      - name: Set up Python 3.7
-        uses: actions/setup-python@v2
-        with:
-          python-version: 3.7
-      - name: Check out code into the Go module directory
-        uses: actions/checkout@v2
-      - name: Check license header
-        run: make license
```

Comment:
I think the original `make license` can be remove or refactored to use license eye, (p.s. we have Docker image for license-eye)

Reply:
Ok. 

By the way, should I remove the python script in `tools/`, which is used to check license headers?

Reply:
> Ok.
> 
> By the way, should I remove the python script in `tools/`, which is used to check license headers?

I suggest to remove it.